LICENSE.md

```diff
@@ -0,0 +1,26 @@
+The MIT License (MIT)
```

Comment:
This change is not really necessary: the `LICENSE` file has no extension by default

Reply:
https://github.com/IQAndreas/markdown-licenses

Reply:
The MIT license doesn't have any formatting anyways so I think its quite pointless

Reply:
The title has heading formatting.